src/components/SkillsBadge/index.jsx

```diff
@@ -4,7 +4,7 @@ const SkillsBadge = ({itemSkills}) => {
     return (
         <>
             {
-                itemSkills.map((skill) => {
+                itemSkills?.map((skill) => {
```

Comment:
do we need the ? anymore?

Reply:
Yes. Furthermore, I have moved conditional rendering logic into the reused components i.e. Salary.